conda-recipe/conda_build_config.yaml

```diff
@@ -5,7 +5,7 @@ c_stdlib:
 - macosx_deployment_target # [osx]
 c_stdlib_version:
 - '2.17' # [linux]
-- '14.4' # [osx]
+- '13.0' # [osx]
```

Comment:
can you pick a valid one from https://anaconda.org/conda-forge/macosx_deployment_target_osx-64/files

instead